xml/geo_manage_i.xml

```diff
@@ -28,130 +28,144 @@
+       </formalpara>
+      </listitem>
+      <listitem>
+       <formalpara>
+        <title>Manual Tickets</title>
+        <para>When using <emphasis>manual</emphasis> tickets, <command>booth
+         grant -F</command> makes booth grant the ticket immediately to the
+         specified site.</para>
+       </formalpara>
+      </listitem>
+     </itemizedlist>
+     <warning>
+      <title>Potential Loss of Data</title>
+      <para>Before using <command>booth grant -F</command>, make sure that no
+       other site (which is online) owns the same ticket. If the same ticket is
+       granted to multiple sites, resources depending on the ticket might start
+       on several sites in parallel. This results in concurrency violation and
+       potential data corruption.</para>
+      <para>As &geo; cluster administrator, you need to resolve a ticket
+       conflict once the other site is reachable again.</para>
+     </warning>
     </listitem>
    </varlistentry>
   </variablelist>
-
-  <para>
-   The <command>grant</command> and (under certain circumstances),
-   <command>revoke</command> operations may take a while to return a definite
-   operation's outcome. The client waits for the result up to the ticket's
-   <varname>timeout</varname> value before it gives up waiting. If the
-   <option>-w</option> option was used, the client will wait indefinitely
-   instead. Find the exact status in the log files or with the
-   <command>crm_ticket -L</command> command.
-  </para>
-
-  <warning>
-   <title><command>crm_ticket</command> and <command>crm&nbsp;site&nbsp;ticket</command></title>
-   <para>
-    If the booth service is not running for any reasons, you may also manage
-    tickets manually with <command>crm_ticket</command> or
-    <command>crm&nbsp;site&nbsp;ticket</command>. Both commands are only
-    available on cluster nodes. Use them with great care as they
-    <emphasis>cannot</emphasis> verify if the same ticket is already granted
-    elsewhere. For more information, read the man pages.
-   </para>
-   <para>
-    As long as booth is up and running, only use the
-    <command>booth&nbsp;client</command> for manual intervention.
-   </para>
-  </warning>
-
-  <para>
-   After you have initially granted a ticket to a site, the booth mechanism
-   takes over and manages the ticket automatically. If the site holding a
-   ticket should be out of service, the ticket is automatically revoked after
-   the expiry time and granted to another site. The resources that depend on
-   that ticket fail over to the new site that holds the ticket. The
-   <literal>loss-policy</literal> set within the constraint specifies what
-   happens to the nodes that have run the resources before.
-  </para>
-
-<!--taroth 2013-04-24: fix for bnc#752601, c#10-->
-
-  <procedure xml:id="pro.ha.geo.manage.tickets">
-   <title>Managing Tickets Manually</title>
+  <para>In the following, find some examples for managing tickets in different
```

Comment:
"In the following"  ?  maybe better "find some example  below" ? or "in the next paragraph"